workspaces/cli-server/src/analytics.ts

```diff
@@ -19,6 +19,9 @@ import {getOrCreateAnonId} from "@useoptic/cli-config/build/opticrc/optic-rc";
 const packageJson = require('../package.json');
 
 const clientId = `local_cli_${packageJson.version}`;
+const platform = os.platform();
```

Comment:
Looks good. See also: `workspaces/cli-scripts/src/errors.ts` for what we send to Sentry

Reply:
Thanks, it appears that Sentry already grabs some of this, so I added the host's release information as well.